src/lib/datepicker/_datepicker-theme.scss

```diff
@@ -33,8 +33,9 @@ $mat-calendar-weekday-table-font-size: 11px !default;
     border-top-color: mat-color($foreground, icon);
   }
 
-  .mat-calendar-next-button,
-  .mat-calendar-previous-button {
+  .mat-datepicker-toggle,
+  .mat-calendar-next-button::after,
```

Comment:
It was being overwritten by the `.mat-icon-button` if I target the parent. I think it depends on the order in which the style tags were added.

Reply:
I would just bump the specificity on the rule then, rather than just targeting the pseudo-element. The button may use the color to determine the color for ripples and things

Reply:
I wanted to avoid increasing the specificity, but I see your point that there could be other things depending on it. It's changed now.